inputremapper/gui/components/editor.py

```diff
@@ -800,7 +800,7 @@ def _on_mapping_message(self, mapping: MappingData):
             self._gui.set_opacity(0.5)
 
         with HandlerDisabled(self._gui, self._on_gtk_changed):
-            self._gui.set_value(mapping.rel_xy_max_input)
+            self._gui.set_value(mapping.rel_input_cutoff)
```

Comment:
so you prefer input_cutoff over max_input?

Reply:
This parameter is probably only ever touched when mapping rel-to-abs and yes, I think input_cutoff communicates better what the parameter does in this case.

It is no like this is the maximum speed at which you can whip the mouse around, it is only the speed at which we ignore higher values for the rel-to-abs handler.